tests/FSharp.Compiler.ComponentTests/EmittedIL/CCtorDUWithMember/CCtorDUWithMember01a.fs.RealInternalSignatureOff.il.net472.bsl

```diff
@@ -357,52 +344,72 @@
       IL_000b:  ret
     } 
 
-    .method public hidebysig virtual final 
```

Comment:
Why is it not virtual anymore?

Reply:
Nono, it still is, it's just that the new overload is inserted prior to this in the generated code and messes up with the diff a bit. The new overload is not virtual, it's not related to any interfaces.